docs/content/TextInput.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: TextInput
+status: Alpha
+---
+
+import {PropsTable} from '../src/props-table'
```

Comment:
You can make PropsTable globally available in markdown files by exporting it here: https://github.com/primer/react/blob/main/docs/src/%40primer/gatsby-theme-doctocat/mdx-components.js

Reply:
It looks like when I try that, I get this error:

<img width="2211" alt="CleanShot 2021-11-30 at 14 52 53@2x" src="https://user-images.githubusercontent.com/21195/144141269-b89e4dcd-762a-433f-bf67-ef55c3968f0c.png">